grouper/entities/pagination.py

```diff
@@ -62,3 +62,9 @@ class ListPermissionsSortKey(Enum):
     NONE = "none"
     NAME = "name"
     DATE = "date"
+
+
+class PermissionGrantSortKey(Enum):
```

Comment:
I do still think this is a bit unfortunate -- namely it makes it possible to have a sort key of "group" when actually looking at service account permission grants or vice versa -- but not the biggest deal. How about putting a comment here noting that this may get used for requests for only one of group grants / service account grants (and thus the value might not always make sense)?

Reply:
I've split the sortkey to avoid this problem